lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -31,6 +31,8 @@ export default (accessToken, userID, requestID) => {
     ),
     saleArtworksLoader: gravityLoader(id => `sale/${id}/sale_artworks`),
     suggestedArtistsLoader: gravityLoader("me/suggested/artists", {}, { headers: true }),
+    myFollowedArtistsLoader: gravityLoader("me/follow/artists", {}, { headers: true }),
+    myFollowedGenesLoader: gravityLoader("me/follow/genes", {}, { headers: true }),
```

Comment:
Why do you feel this needs a `my` prefix? The user specific loaders will always be about the authenticated user anyways.